### PR TITLE
Clean the tooling:

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**
- remove AMD
- remove karma
- use `import * from ""`
- use browserify

---
This change is [Reviewable](https://reviewable.io/reviews/foretagsplatsen/klassified/55)

### CODE REVIEW

**Comment by DamienCassou** [comment]
Do you plan to make that build at some point? :-)

**Comment by BenjaminVanRyseghem** [comment]
I didn't get any notification

**Comment by BenjaminVanRyseghem** [comment]
and as far as I remember, it was already failing

**Comment by BenjaminVanRyseghem** [comment]
done :smile:

File: package.json [hunk 1]
```diff
@@ -51,18 +51,27 @@
...
 		"gulp-load-plugins": "^1.2.0",
 		"gulp-requirejs-optimize": "^1.2.0",
 		"gulp-sequence": "^0.4.5",
 		"gulp-shell": "^0.5.2",
+		"gulp-sourcemaps": "^2.6.1",
 		"gulp-strip-code": "^0.1.3",
+		"gulp-uglify": "^3.0.0",
 		"jasmine": "^2.4.1",
 		"jasmine-reporters": "^2.1.1",
 		"karma": "^0.13.22",
```
Comment (DamienCassou) [inline package.json]: should you remove this as well?

**Comment by DamienCassou** [comment]
I love it. Thank you!

**Comment by DamienCassou** [comment]
Apparently, this breaks our code: `klassified.object` seems undefined now. Do you have any clue what should be changed on our side?

File: package.json (hunk 1 above)
Comment (BenjaminVanRyseghem) [reply package.json]: yep

**Comment by BenjaminVanRyseghem** [comment]
@DamienCassou I know what should be changed (`klassified.default.object`) but I'm gonna change it as it's too long to me :smile:

**Comment by BenjaminVanRyseghem** [comment]
new commit, it should be fine now

**Pull request merged**